tests/test_gauss_pdf_and_cdf.py

```diff
@@ -0,0 +1,30 @@
+from os.path import exists
+from src.gauss_pdf_and_cdf import (GaussianProbabilisticSetting,
+                                   BetaProbabilisticSetting)
+import cv2
+
+
+def show_or_save_figure(distribution_type):
+    if distribution_type == "gaussian":
+        distribution = GaussianProbabilisticSetting()
+    elif distribution_type == "beta":
+        distribution = BetaProbabilisticSetting()
+
```

Comment:
Add else statement to raise NotImplementedError. Also, tell users to choose from existing distribution_types.

Reply:
Also, tell users to choose from existing distribution_types.